docs/zh/openmldb_sql/ddl/CREATE_TABLE_STATEMENT.md

```diff
@@ -184,6 +200,15 @@ CREATE TABLE db1.t1 LIKE HIVE 'hive://hive_db.t1';
 
 注意目前创建的表名必须包括DB名。
 
+**示例6：基于 Parquet 文件创建新表**
+
+```sql
+CREATE TABLE db1.t1 LIKE PARQUET 'file://t1.parquet';
+-- SUCCEED
+```
+
+注意目前创建的表名必须包括DB名。
```

Comment:
db is not optional if we have `use db`?

Reply:
It is a bug now and we will fix in https://github.com/4paradigm/OpenMLDB/pull/3063 .